src/Lorentz/Contracts/RegistryDAO.hs

```diff
@@ -172,11 +173,51 @@ decisionLambda = do
     )
   nil
 
-config ::
+
+data CustomParam k v
```

Comment:
I think this should be in `Lorentz.Contracts.RegistryDAO.Types`, also to stay in line with the rest of the "unwritten convention" it should have `RegistryDAO` in its name.

Reply:
Have renamed the type and moved it to `Lorentz.Contracts.RegistryDAO.Types` module.